src/dbnode/storage/namespace_readers.go

```diff
@@ -238,9 +266,22 @@ func (m *namespaceReaderManager) get(
 	blockStart time.Time,
 	position readerPosition,
 ) (fs.DataFileSetReader, error) {
+	latestVolume, err := m.latestVolume(shard, blockStart)
+	if err != nil {
+		return nil, err
+	}
+
+	// If requesting an outdated volume, we need to start again reading from
```

Comment:
"again reading" -> "reading again"

Reply:
Can you add more to the comment basically just saying this is ok since its fine to return duplicate metadata?

Reply:
also maybe just say that the cached reader that was for the previous volume will get cleaned up automatically by the ticking process so we dont need to worry about closing it here.